conda/recipes/kvikio/meta.yaml

```diff
@@ -2,7 +2,7 @@
 
 {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
 {% set py_version = environ.get('CONDA_PY', 36) %}
-{% set cuda_version = '.'.join(environ.get('CUDA', '11.5').split('.')[:2]) %}
+{% set cuda_version = '.'.join(environ['RAPIDS_CUDA_VERSION'].split('.')[:2]) %}
```

Comment:
Because of this change, we'll need to add the following workaround variables to both `ci/{cpu,gpu}/build.sh` temporarily to get Jenkins builds working.

- https://github.com/rapidsai/cudf/blob/d1cebdbf50d92358962bc66a1b2c0a8476de1f35/ci/cpu/build.sh#L25-L30

Reply:
As well as `export RAPIDS_DATE_STRING=$(date +%y%m%d)`.

Reply:
update the `ci/{cpu,gpu}/build.sh` scripts